### PR TITLE
[create] Ask prompt if there are no staged changes

### PR DESCRIPTION
**Pull request by costinsin**

### CODE REVIEW

**Comment by costinsin** [comment]
Automatically created using [gi](https://github.com/costinsin/gi).

File: src/commands/create.rs [hunk 1]
```diff
@@ -26,8 +29,23 @@ pub fn create() -> Result<()> {
     }
 
     if !working_area.has_staged_changes() {
-        // TODO: Add a prompt to ask the user if they want to stage all changes
-        git_client.add_all()?;
+        let items = vec!["Commit all changes (--all)", "Abort operation"];
```
Comment (RazvanDire) [inline src/commands/create.rs:32]: I imagine that in the future we'll add something like "Select files to commit"

**Review by RazvanDire — APPROVED** [review]

File: src/commands/create.rs (hunk 1 above)
Comment (costinsin) [reply src/commands/create.rs:32]: Yes, add with `--patch` option. https://nuclearsquid.com/writings/git-add/

**Comment by costinsin** [comment]
### Merge activity

* **May 10, 3:20 AM EDT**: @costinsin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/costinsin/gi/40).
* **May 10, 3:20 AM EDT**: @costinsin merged this pull request with [Graphite](https://app.graphite.dev/github/pr/costinsin/gi/40).

**Pull request merged**